pages/_app.tsx

```diff
@@ -4,13 +4,15 @@ import 'keen-slider/keen-slider.min.css'
 
 import { FC, useEffect } from 'react'
 import type { AppProps } from 'next/app'
-import { Head } from '@components/common'
+import { Head, Layout } from '@components/common'
 import { ManagedUIContext } from '@components/ui/context'
 
-const Noop: FC = ({ children }) => <>{children}</>
+type LayoutT = FC & { Layout: typeof Layout }
 
+const Noop: FC = ({ children }) => <>{children}</>
+cl
 export default function MyApp({ Component, pageProps }: AppProps) {
-  const Layout = (Component as any).Layout || Noop
+  const Layout = (Component as LayoutT).Layout || Noop
```

Comment:
In this case the type `any` is fine, we already know that `Component` is a react component, but the code isn't interested in the value of `Layout` or whether it exists or not (as it has a fallback), and we also don't want to enforce it to be the `Layout` component.